src/client/puppet-service.ts

```diff
@@ -1449,12 +1450,15 @@ export class PuppetService extends Puppet {
 
   public async friendshipAdd (
     contactId : string,
-    hello     : string,
+    options   : FriendshipAddOptions,
   ): Promise<void> {
-    log.verbose('PuppetService', 'friendshipAdd(%s, %s)', contactId, hello)
+    log.verbose('PuppetService', 'friendshipAdd(%s, %s)', contactId, JSON.stringify(options))
 
     const request = new FriendshipAddRequest()
     request.setContactId(contactId)
+    request.setOptions(options)
```

Comment:
This `options` is not valid type here, the `FriendshipAddOptions` should be imported from `@chatie/grpc`.